components/user/UserLink.js

```diff
@@ -38,6 +39,7 @@ export default function UserLink({
     >
       <Icon name={link.icon} />
       <span className="grow">{link.name}</span>
+      <FaEye />
```

Comment:
I think this needs to be in the conditional statement below, otherwise it will show for profiles that don't display their stats